driver/src/gas_station.rs

```diff
@@ -45,8 +46,10 @@ impl GnosisSafeGasStation {
 }
 
 impl GasPriceEstimating for GnosisSafeGasStation {
-    fn estimate_gas_price(&self) -> Result<GasPrice> {
-        self.client.get_json(&self.uri, HttpLabel::GasStation)
+    fn estimate_gas_price(&self) -> BoxFuture<Result<GasPrice>> {
+        self.client
+            .get_json_async(&self.uri, HttpLabel::GasStation)
```

Comment:
I think we should be able to remove `get_json` now in a separate PR.

Reply:
Turns out I have to include this here anyway or it won't compile due to dead code.